.github/workflows/e2e_nightly_upgrade.yml

```diff
@@ -0,0 +1,204 @@
+name: Nightly upgrades with user inferfaces
+
+on:
+  push:
+  schedule:
+    - cron: "0 6 * * *" # Every day at 06:00Am
+
+jobs:
+  # Pre-job : Get Upgrade target versions
+  get_ps_target_versions:
+    name: Get Upgrade target versions
+    runs-on: ubuntu-18.04
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Setup Node
+        uses: actions/setup-node@v2
+        with:
+          node-version: 10
+
+      - name: Install dependencies
+        shell: bash
+        run: npm install
+        working-directory: ./tests/e2e
+
+      - id: set-target-versions
+        shell: bash
+        run: echo "::set-output name=TARGET_VERSIONS::$(node ./scripts/getTargetVersions.js)"
+        env:
+          API_URL: https://api-nightly.prestashop.com/reports
+        working-directory: ./tests/e2e
+
+    outputs:
+      target_versions: ${{ steps.set-target-versions.outputs.TARGET_VERSIONS }}
```

Comment:
use env instead https://docs.github.com/en/actions/reference/workflow-commands-for-github-actions#setting-an-environment-variable

Reply:
I did the modification https://github.com/PrestaShop/autoupgrade/pull/387/commits/ef3122bdad9237833bb2bd88f0973e7c1f7795f7 but I got a context error `Unrecognized named-value: 'env'.` So I deleted it.

Reply:
Because it should be only one job, they are link, one can't work without the other.